xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -169,8 +169,8 @@ final class ClientXdsClient extends XdsClient implements XdsResponseHandler, Res
           || Boolean.parseBoolean(System.getenv("GRPC_EXPERIMENTAL_XDS_CUSTOM_LB_CONFIG"));
   @VisibleForTesting
   static boolean enableOutlierDetection =
-      !Strings.isNullOrEmpty(System.getenv("GRPC_EXPERIMENTAL_XDS_OUTLIER_DETECTION"))
-          || Boolean.parseBoolean(System.getenv("GRPC_EXPERIMENTAL_XDS_OUTLIER_DETECTION"));
+      !Strings.isNullOrEmpty(System.getenv("GRPC_EXPERIMENTAL_ENABLE_OUTLIER_DETECTION"))
```

Comment:
Looking at the other env var names used we are not consistent in using "ENABLE" in the name. Is ENABLE the new thing or not having ENABLE is the new thing?

Reply:
Yeah, we are not consistent with all of these, but this is what the outlier detection gRFC specifies, so that's what I'm going with here.